app/views/inventories/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>My Inventories</h1>
+
+<ul>
+  <% @inventories.each do |inventory| %>
+    <li>
+      <%= link_to inventory.name, inventory %>
+      <%= button_to 'Delete', inventory, method: :delete, data: { confirm: 'Are you sure?' } if inventory.user == current_user %>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [ ] [REQUIRED] Great effort so far team, however, we need to add the description and style the inventory display list created by the logged-in user as stated by the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="740" alt="Screenshot 2023-09-16 at 22 51 06" src="https://github.com/techmoves/recipe-app/assets/58692590/eb185832-9b86-4f9e-81e5-b9139e833277"> | <img width="919" alt="Screenshot 2023-09-16 at 22 51 29" src="https://github.com/techmoves/recipe-app/assets/58692590/78902631-b876-4af9-b261-e257e19a6c1d">